Moya.xcodeproj/project.pbxproj

```diff
@@ -963,20 +963,24 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				"FRAMEWORK_SEARCH_PATHS[sdk=appletv*]" = (
-					"$(SRCROOT)/Carthage/Build/tvOS/",
 					"$(inherited)",
+					"$(SRCROOT)/Carthage/Build/",
+					"$(SRCROOT)/Carthage/Build/tvOS",
 				);
 				"FRAMEWORK_SEARCH_PATHS[sdk=iphone*]" = (
-					"$(SRCROOT)/Carthage/Build/iOS/",
 					"$(inherited)",
+					"$(SRCROOT)/Carthage/Build/",
```

Comment:
why did you have to add both this line and `"$(SRCROOT)/Carthage/Build/iOS"`?

Reply:
not sure if you did this on purpose, or if it's a default from Xcode, but maybe you can remove this line? looks like it happened for every scheme other than the tvOS one

Reply:
I did it on purpose to maintain compatibility for building old style framework bundle.

I can try for a more elegant solution if possible

Reply:
ahh that makes sense. no that's fine then, didn't realize that was the old style.iI've seen it before, i thought it was some Xcode default behavior. will merge after i test